slides/08-Platform-Hub/1070-WhatDoYouThink.md

```diff
@@ -0,0 +1,5 @@
+## We love feedback
+
+Please give us feedback, this is a realtively new application so there's going to be teathing issues. However, we are driven to sorting as many of them as we can and improving how projects interact with acp platform. The following is the best place to make new issues https://github.com/UKHomeOffice/platform-hub/issues
```

Comment:
~~teathing~~teething
~~acp~~ACP

email? `mailto:....` link
slack? what channel ? link to it